core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -262,20 +267,20 @@ protected function row($strPalette=null)
 		}
 
 		// Input field callback
-		if (\is_array($arrData['input_field_callback']))
+		if (\is_array($arrData['input_field_callback'] ?? null))
 		{
 			$this->import($arrData['input_field_callback'][0]);
 
 			return $this->{$arrData['input_field_callback'][0]}->{$arrData['input_field_callback'][1]}($this, $xlabel);
 		}
 
-		if (\is_callable($arrData['input_field_callback']))
+		if (\is_callable($arrData['input_field_callback'] ?? null))
 		{
 			return $arrData['input_field_callback']($this, $xlabel);
 		}
 
 		/** @var Widget $strClass */
-		$strClass = $GLOBALS['BE_FFL'][$arrData['inputType']];
+		$strClass = $GLOBALS['BE_FFL'][$arrData['inputType']] ?? null;
```

Comment:
```suggestion
		$strClass = $GLOBALS['BE_FFL'][$arrData['inputType']] ?? '';
```

Because `class_exists` expects a string.